test/reader-test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fr = require('../index.js');
+
+describe('File Reader Module', () => {
+  describe('following proper file paths', () => {
+    it('should have working path \"__dirname\"/assets/file', (done) => {
+      fr.fileRunner((err) => {
+        expect(err).to.not.throw();
+        done();
+      });
+    });
+  });
+  describe('logging in correct order', () => {
+    it('Should log one.txt then two.txt then three.txt', (done) => {
+      fr.fileRunner((err) => {
+        expect(err).to.equal('null');
```

Comment:
the string 'null' is different from the keyword null; Also you are passing in a callback function to a function that has a first parameter of path.Also you want this function to be able to accept some sort of Data back so that you can actually test that.